db/src/main/resources/db/migration/V13__datastream_grants.sql

```diff
@@ -0,0 +1,23 @@
+DO
+$$
+BEGIN
+        IF EXISTS
+            (SELECT 1 from pg_roles where rolname = 'im-db')
+        THEN
+            ALTER USER "im-db" WITH REPLICATION;
+END IF;
+END
+$$;
+DO
+$$
+BEGIN
+        IF EXISTS
+            (SELECT 1 from pg_roles where rolname = 'simba_datastream_bruker')
+        THEN
+            ALTER USER "simba_datastream_bruker" WITH REPLICATION;
+            ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON TABLES TO "simba_datastream_bruker";
+GRANT USAGE ON SCHEMA public TO "simba_datastream_bruker";
+            GRANT SELECT ON ALL TABLES IN SCHEMA public TO "simba_datastream_bruker";
```

Comment:
Gjør ikke linje 18 og 20 det samme?

Reply:
Jeg er ikke sikker men tror ikke det. 
Det er bare copy pastet fra prosjekt til prosjekt så kan hende det går ann å gjøre det annerledes.

Reply:
Etter et kjapt google søk så kan det virke som det er nødvendig. 
Fordi alter default bare gjelder for fremtitige tabeller. Mens man også må legge det til å eksisterende.